### DISCUSSION TITLE
changes do not get updated

### DISCUSSION THREAD

**Discussion by ogorei** [GitbookIO/community#522] [0. General bugs and questions]
Have made multiple changes but none are showing on the public link website.
status
1. Account synced ✅
2. new commits merged
3. main branch updated to latest commits

Could anyone provide answers on how to solve this?

**Comment by aikoii** [comment]
👋 @ogorei, we've rolled out a couple of fixes last week for the delay in content updates due to caching issues. A [hard refresh](https://docs.gitbook.com/faq/hard-refresh) should show the changes right away. Let me know if you still have trouble.
